src/services/jwt.service.js

```
@@ -0,0 +1,38 @@
+import jwt from 'jsonwebtoken';
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '30d',
+  });
+
+  return token;
+}
+
+function verify(token) {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
```

verify and verifyRefresh swallow errors and return null; consider logging the caught exception at debug level to aid troubleshooting (avoid leaking sensitive info in logs). Returning null is fine, but silent failures can make debugging harder in development.